R/tm_front_page.R

```diff
@@ -77,7 +77,7 @@ tm_front_page <- function(label = "Front page",
     ui = ui_front_page,
     ui_args = args,
     server_args = list(tables = tables, show_metadata = show_metadata),
-    filters = NULL
+    filters = if (show_metadata) "all" else NULL
```

Comment:
I don't like this, but if I don't do this I get an error from teal saying if filters = NULL then you cannot use data